data/pathfinder/paizo/player_companion/animal_archive/anar_abilities_class.lst

```diff
@@ -29,6 +29,51 @@ Pack Attack				KEY:Huntmaster ~ Pack Attack				CATEGORY:Special Ability	TYPE:Ext
+Animal Companion / XVIII	KEY:Huntmaster ~ Animal Companion 18	CATEGORY:Special Ability	TYPE:HuntmasterCompanionChoice																																							PREVARGTEQ:AnimalCompanionLVLQ,1	PREVARLT:AnimalCompanionLVLR,CavalierLVL	DEFINE:AnimalCompanionLVLR|0		DESC:Total Levels Assigned to Companion %1|AnimalCompanionLVLR																																																																																																																																																																																																																																																	STACK:YES	MULT:YES	CHOOSE:NOCHOICE	ABILITY:Special Ability|AUTOMATIC|Huntmaster Follower ~ 18																														BONUS:VAR|AnimalCompanionLVLR|1																																																																								SOURCEPAGE:p.125
+Animal Companion / XIX		KEY:Huntmaster ~ Animal Companion 19	CATEGORY:Special Ability	TYPE:HuntmasterCompanionChoice																																							PREVARGTEQ:AnimalCompanionLVLR,1	PREVARLT:AnimalCompanionLVLS,CavalierLVL	DEFINE:AnimalCompanionLVLS|0		DESC:Total Levels Assigned to Companion %1|AnimalCompanionLVLS																																																																																																																																																																																																																																																	STACK:YES	MULT:YES	CHOOSE:NOCHOICE	ABILITY:Special Ability|AUTOMATIC|Huntmaster Follower ~ 19																														BONUS:VAR|AnimalCompanionLVLS|1																																																																								SOURCEPAGE:p.125
+Animal Companion / XX		KEY:Huntmaster ~ Animal Companion 20	CATEGORY:Special Ability	TYPE:HuntmasterCompanionChoice																																							PREVARGTEQ:AnimalCompanionLVLS,1	PREVARLT:AnimalCompanionLVLT,CavalierLVL	DEFINE:AnimalCompanionLVLT|0		DESC:Total Levels Assigned to Companion %1|AnimalCompanionLVLT																																																																																																																																																																																																																																																	STACK:YES	MULT:YES	CHOOSE:NOCHOICE	ABILITY:Special Ability|AUTOMATIC|Huntmaster Follower ~ 20																														BONUS:VAR|AnimalCompanionLVLT|1																																																																								SOURCEPAGE:p.125
+
+# AUTO ADDED ABILITIES:
```

Comment:
Is this section incomplete? I do not see any reference to these abilities, nor any obvious reason for their existence.